src/model.c

```diff
@@ -570,7 +570,7 @@ struct model *model_load_from_stl(const char *fname)
     // For every 3 vertices create a face
     for (int i = 0; i < model->vertex_count; i += 3)
     {
-        model_add_face(model, i, i+1, i+2, current_material);
+        model_add_face(model, i, i+2, i+1, current_material);
```

Comment:
I would squash this patch with the previous one, since this flipping of the face directions is required exactly when the axes are swapped.